src/stock-user-role-scopes/add-stock-user-scope/add-stock-user-role-scope.scss

```diff
@@ -0,0 +1,17 @@
+@use '@carbon/styles/scss/spacing';
+@use '@carbon/styles/scss/type';
+@import '~@openmrs/esm-styleguide/src/vars';
```

Comment:
Nit: We advise folks to prefer Carbon color tokens over the variables declared in `_vars.scss`. At some point in the future, we'll get rid of `_vars.scss`, so moving forward use this [tool](https://carbon-elements.netlify.app/colors/examples/preview/) to find the correct token to use and then substitute accordingly in your stylesheet. Read the [styling](https://o3-docs.openmrs.org/docs/coding-conventions#styling) guide in the Coding Conventions for inspiration.